electrum_dash/plugins/trezor/trezor.py

```diff
@@ -207,7 +207,7 @@ def get_client(self, keystore, force_pair=True, *,
         return client
 
     def get_coin_name(self):
-        return "Zcoin Testnet" if constants.net.TESTNET else "Zcoin"
+        return "FIRO Testnet" if constants.net.TESTNET else "FIRO"
```

Comment:
This should be lower case:

```suggestion
        return "Firo Testnet" if constants.net.TESTNET else "Firo"
```